mocks/dynamo.go

```diff
@@ -167,289 +162,28 @@ ItemLoop:
 	return &dynamodb.GetItemOutput{Item: match}, nil
 }
 
-// ScanPages mocks the dynamo ScanPages method
-func (d *MockDynamo) ScanPages(in *dynamodb.ScanInput, pager func(*dynamodb.ScanOutput, bool) bool) (err error) {
```

Comment:
Deleting scan mock, because...it looks like it was only used in tests of a mock in `mocks/dynamo_test.go` ? I don't want to tests mock :p 

Get/Put/Update seem to be also used in `checkpoints_test.go` with mock